modules/administration/pages/monitoring.adoc

```diff
@@ -1,26 +1,23 @@
-// 2019-07-24, ke: Is it supported on Salt minions only?
-// This seems to indicate it: "Installing and configuring exporters is done using a Salt formula."
-// If yes, we'd better replace "mananged clients" with the term "Salt minions".
-
-[[prometheus]]
+[[monitoring]]
 = Monitoring with Prometheus
 
-Monitoring can be performed in {productname} using Prometheus and Grafana.
-The packages for Prometheus are shipped with SUSE Manager Client Tools, these packages also include several Prometheus exporters.
-{productname} Server and Proxy are now able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on managed client systems.
+You can monitor your {productname} environment using Prometheus and Grafana.
+{productname} Server and Proxy are able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on Salt clients.
 
-// I do not see why we should mention where this was started; we do not do this elsewhere
 Prometheus is a monitoring tool that is used to record real-time metrics in a time-series database.
-Unlike other monitoring systems, Prometheus collects metrics using HTTP pulls, allowing for higher performance and scalability.
-Prometheus is an open-source software project, mostly written in Go.
-Its source code is available at https://github.com/prometheus/.
+It is an open-source software project, written in Go.
+Metrics are collected using HTTP pulls, allowing for higher performance and scalability.
+For more information about Prometheus, see https://prometheus.io/docs/.
 
 Grafana is a tool for data visualization, monitoring, and analysis.
 It is used to create dashboards with panels representing specific metrics over a set period of time.
 Grafana is commonly used together with Prometheus, but also supports other data sources such as ElasticSearch, MySQL, PostgreSQL, and Influx DB.
 For more information about Grafana, see https://grafana.com/docs/.
 
-A Prometheus package is included in the SUSE Manager Client Tools for {sle}{nbsp}12 and {sle}{nbsp}15.
+You need to install Prometheus and Grafana on a machine separate from the {productname} Server.
+We recommend you use a managed client as your monitoring server.
```

Comment:
Here, the explicit instruction that Salt Client is an example of manager client van be added, given that it (Salt Client) appears further down in the text.